src/NovaTwoFactor.php

```diff
@@ -34,29 +35,42 @@ public function menu(Request $request)
         }
     }
 
-    public static function promptEnabled(Request $request)
+    public static function promptEnabled(Request $request): bool
     {
 
         $timeout = config('nova-two-factor.reauthorize_timeout', 5);
 
-        $promptFor = config('nova-two-factor.reauthorize_urls', []);
+        $promptFor = array_map(fn ($el) => trim(Nova::url($el), '/'), config('nova-two-factor.reauthorize_urls', []));
 
         $hasUrl = $request->is($promptFor);
 
-        $lastAttempt = session()->get('2fa.prompt_at', now()->subMinutes($timeout + 1));
+        $lastAttempt = self::getLastPromptTime();
+
+        if ($lastAttempt->diffInMinutes(now()) >= $timeout && $hasUrl) {
 
-        if ($lastAttempt->diffInMinutes(now()) > $timeout && $hasUrl) {
             return true;
         }
 
         return false;
     }
 
-
     public static function prompt()
     {
         return inertia('NovaTwoFactor.Prompt', [
-            'referer' => request()->url()
+            'referer' => request()->url(),
         ]);
     }
+
+    public static function setLastPromptTime(): void
+    {
+        session()->put('2fa.prompt_at', now());
+    }
+
+    public static function getLastPromptTime()
+    {
+        $timeout = config('nova-two-factor.reauthorize_timeout', 5);
+
+        return session()->get('2fa.prompt_at', now()->subMinutes($timeout + 5));
+        Session::put('2fa.prompt_at', now());
```

Comment:
this is never called

Reply:
Yes. my bad conflict resolution went wrong. will fix soon